core-bundle/contao/forms/Form.php

```diff
@@ -560,10 +587,41 @@ protected function processFormData($arrSubmitted, $arrLabels, $arrFields, $arrFi
 			System::getContainer()->get('monolog.logger.contao.forms')->info('Form "' . $this->title . '" has been submitted by a guest.');
 		}
 
+		$targetPageData = null;
+
 		// Check whether there is a jumpTo page
 		if (($objJumpTo = $this->objModel->getRelated('jumpTo')) instanceof PageModel)
 		{
-			$this->jumpToOrReload($objJumpTo->row());
+			$targetPageData = $objJumpTo->row();
+		}
+
+		// Set the confirmation message, if any
+		if ($this->objModel->confirmationMessage)
+		{
+			$confirmationMessage = $this->objModel->confirmationMessage;
+			$confirmationMessage = System::getContainer()->get('contao.string.simple_token_parser')->parse($confirmationMessage, array_map(array(StringUtil::class, 'specialchars'), $arrSubmitted));
```

Comment:
```suggestion
			$confirmationMessage = System::getContainer()->get('contao.string.simple_token_parser')->parse($confirmationMessage, array_map(StringUtil::specialchars(...), $arrSubmitted));
```

As Contao 5 requires PHP 8.1 we can use [first class callables](https://www.php.net/manual/en/functions.first_class_callable_syntax.php) here ☺️